src/omniperf_base.py

```diff
@@ -51,6 +51,9 @@ def __init__(self):
             "gfx906": {"mi50": ["MI50", "MI60"]},
             "gfx908": {"mi100": ["MI100"]},
             "gfx90a": {"mi200": ["MI210", "MI250", "MI250X"]},
+            "gfx940": {"mi300": ["MI300A_A0"]},
+            "gfx941": {"mi300": ["MI300X_A0"]},
+            "gfx942": {"mi300": ["MI300A_A1", "MI300X_A1"]},
```

Comment:
wahoo

Reply:
This format maps arch to series and series to variants. This is helpful because in, let's say, Mi300 when several architectures map to the same series, we can reserve the option to use common metric config for the entire series.

For now, we're maintaining individual configs for each architecture, but this could be useful for future enhancements